### ISSUE TITLE
Feature Request: Clarify Action Item for Channel Update Warnings

### ISSUE THREAD

**Issue by zapomatic** [C-Otto/lnd-manageJ#92]
**Is your feature request related to a problem? Please describe.**

My dashboard is full of warnings like these, calling out that the channel has accumulated too many updates, but the UI and docs do not provide any insight into what could/should be done to resolve this. Is this more of an information note than an actionable warning? If so, it may not be useful to stack up on the top of the dashboard since I have so many channels with this warning that it takes up the entire above the fold content area of the dashboard. If it is actionable (aside from closing the channel), then providing a tool or insight to take action would be helpful.

<img width="293" alt="Screenshot 2024-02-14 at 8 31 01 AM" src="https://github.com/C-Otto/lnd-manageJ/assets/155849471/b43457dc-cff2-4c40-983e-595a91c1d066">

**Describe the solution you'd like**
Explain in the docs what this means, and if there is an action item that a node runner can take, provide details.

**Comment by C-Otto** [comment]
I set the warning threshold to 1,000,000 on my node (the default is 100,000) and I agree that it's not as important as it was "back in the days". Previously, the `channel.db` was around 27x as large and the updates were the main reason for this. I think it's time to remove the warning, as the issue has been fixed with lnd 0.15.1 and most node runners don't really see any issue related to database file sizes anymore.

See https://bitcoin.stackexchange.com/questions/110331/why-is-lnds-channel-db-so-huge-and-when-will-it-stop-growing
